### ISSUE TITLE
Exception: Start may not be called on a continuation task

### ISSUE THREAD

**Issue by JeremyBP** [dansiegel/Prism.Container.Extensions#118]
I'm getting this exception when using ExecutionAwareCommand.FromTask< MyObject >(MyMethod);
No problem with ExecutionAwareCommand.FromTask(MyMethod).
So I digged into the source code of the ExecutionAwareCommand< T > class and found a [Start()](https://github.com/dansiegel/Prism.Container.Extensions/blob/dfab08005cf36f5660c7420acb678b0d9aad7f0c/src/Prism.Forms.Extended/Commands/ExecutionAwareCommand%7BT%7D.cs#L80) call wich throws the exception.
Then, if I just remove the Start call like it is into the ExecutionAwareCommand, no more exception and task just runs like it should...
Any reason of this Start call that I miss?

**Comment by dansiegel** [comment]
can you provide a reproduction of the issue?

**Comment by JeremyBP** [comment]
I started with the fresh latest master commit.
On the provided sample, I just added a button on the MainPage like so:

`<Button Text="Command with param" Command="{Binding WithParamCommand}" CommandParameter="p1"/>`

Then on the MainPageViewModel :

`public ICommand WithParamCommand { get; }`

`private async Task WithParamAsync(string param)`
`{`
    `// Whatever awaitable stuff here doesn't matter`
`}`

and finally into the construtor:

`WithParamCommand = ExecutionAwareCommand.FromTask<string>(WithParamAsync);`

From there, clicking on the button throws the InvalidOperationException: Start may not be called on a continuation task.

If you remove the [`.Start();`](https://github.com/dansiegel/Prism.Container.Extensions/blob/dfab08005cf36f5660c7420acb678b0d9aad7f0c/src/Prism.Forms.Extended/Commands/ExecutionAwareCommand%7BT%7D.cs#L80) call, then no more exception.

Maybe I'm doing something wrong or missing something.
